compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -503,26 +506,23 @@ void addConvTileAndDecomposeExpertPassPipeline(OpPassManager &passManager,
 
   nestedModulePM.addNestedPass<func::FuncOp>(createLLVMCPUTileAndFusePass(
       static_cast<int64_t>(TilingLevel::ParallelTiles)));
-  if (clEnablePadConsumerFusion) {
```

Comment:
Are we able to drop the option entirely? If so, let's remove its definition, i.e., line 46-53 in this file.

Reply:
Yes, I think so. I was planning to do that as a follow up.